src/main/java/org/randywebb/wlts/beans/AbstractBean.java

```diff
@@ -74,17 +74,17 @@ See setFromJSON() for how subclasses can handle non-string values in the JSON
 
                 for (Object o : array) {
                     try {
-                        toFill.add(constructor.newInstance(new Object[] {(JSONObject)o}));
-                    } catch(InvocationTargetException e) {
+                        toFill.add(constructor.newInstance(new Object[] {(JSONObject) o}));
+                    } catch (InvocationTargetException e) {
                         log.error("Cannot invoke " + clazz.getName() + "(JSONObject)");
-                    } catch(IllegalAccessException e) {
+                    } catch (IllegalAccessException e) {
                         log.error("No permissions to create " + clazz.getName() + "(JSONObject)");
-                    } catch(InstantiationException e) {
+                    } catch (InstantiationException e) {
                         log.error("Unable to create " + clazz.getName() + "(JSONObject)");
                     }
                 }
 
-            } catch(NoSuchMethodException e) {
+            } catch (NoSuchMethodException e) {
                 log.error("No constructor " + clazz.getName() + "(JSONObject)");
```

Comment:
It's a common thing that I see and is usually unintentional. I think I see where you've used the second argument in most other places in the code, so I wasn't sure if this was an oversight or on purpose.
I don't require that we pass the exception in log statements, but wanted to confirm that this was intentional vs oversight.
FYI, I often see people do: log.error("Message" + e.toString()) which really doesn't work for them as e.toString() usually just writes out a memory address instead of the error message :)